### PR TITLE
[FSDE-1479] Added support of chiSelectAll & chiDeselectAll event emission

### PR DESCRIPTION
**Pull request by janibolkvadze**
https://ctl.atlassian.net/browse/FSDE-1479

### CODE REVIEW

**Comment by SCMAUTO** [comment]
The CI pipeline run successfully in https://jenkinsprod.corp.intranet:8443/job/UX-CHI/job/Productive/job/Chi/job/PR-132/1/. :white_check_mark:

**Comment by SCMAUTO** [comment]
You can check this PRs instance in https://f2edc23299318b2d5e9bafd130e07978aa93a6f0-chi-pr.enterprise.ctl.io

File: src/chi-vue/src/views/DataTable/ClientSide/fixtures.ts [hunk 1]
```diff
@@ -8,7 +8,7 @@ export const exampleConfig = {
     bordered: false,
     hover: false,
     size: 'lg',
-    striped: true,
+    striped: false,
```
Comment (dani-cl-madrid) [inline src/chi-vue/src/views/DataTable/ClientSide/fixtures.ts:11]: Is this change expected?

File: test/chi/chi-vue/data-table-portal.pug [hunk 2]
```diff
@@ -42,7 +42,7 @@ script.
           bordered: false,
           hover: false,
           size: "md",
-          striped: false,
+          striped: true,
```
Comment (dani-cl-madrid) [inline test/chi/chi-vue/data-table-portal.pug:45]: Same here. Is this change expected?

**Review by dani-cl-madrid — CHANGES_REQUESTED** [review]

File: test/chi/chi-vue/data-table-portal.pug (hunk 2 above)
Comment (janibolkvadze) [reply test/chi/chi-vue/data-table-portal.pug:45]: Yes, UX/UI requested making the striping optional.
Previously enabling `portal: true`, striping was automatic.
This is a test page .pug file. The test was failing as now the striping is optional ;)

**Review by dani-cl-madrid — APPROVED** [review]

**Pull request merged**